Docs - add feature docs explaining how to use the library

stijnmoreels opened the pull request:
Closes #4

stijnmoreels commented:
@tomkerkhove , is there something wrong with the docs? Like, a syntax error or so? Or is it bc there's no pre-section with `--- style: default ---`?

mbraekman commented on docs/preview/README.md:
````diff
@@ -0,0 +1,17 @@
+Easy to install it via NuGet.
+
+```shell
+PM > Install-Package Invictus-Testing
````
_Perhaps this remark is a bit late in the process, but still.._
Shouldn't the name be more specific and point towards Logic Apps, instead of `Invictus-Testing`.
Or wouldn't we add any other test-framework under the Invictus-umbrella?

tomkerkhove replied:
That's a fair point, maybe renaming to `Invictus.Testing.LogicApps` for now is best then?

mbraekman replied:
Yes, let's rename it while it isn't used yet.
You never know whether other test-frameworks would be added later on.

stijnmoreels replied:
➡️ https://github.com/invictus-integration/testing-framework/issues/44

tomkerkhove replied:
Ha, will close #43 😄

stijnmoreels commented:
> Can you:

> * Configure the sidebar navigation?

Added a link here.

> * Move it from preview to the final docs for now since it's our first version?

Hmm, it's placed under `./docs/preview` bc it's not yet fully released yet, not bc it is the first version.

> * Move `README.md` to `Home.md`

Yes, renamed that.

tomkerkhove commented:
> > * Move it from preview to the final docs for now since it's our first version?
> 
> Hmm, it's placed under `./docs/preview` bc it's not yet fully released yet, not bc it is the first version.

Problem is that it won't work with this model of docs as it's not using the same approach as with Arcus (see preview)

stijnmoreels commented:
> > > * Move it from preview to the final docs for now since it's our first version?
> > 
> > 
> > Hmm, it's placed under `./docs/preview` bc it's not yet fully released yet, not bc it is the first version.
> 
> Problem is that it won't work with this model of docs as it's not using the same approach as with Arcus (see preview)

Ok, done that.

tomkerkhove commented on docs/_sidebar.md:
```diff
@@ -1,3 +1,6 @@
 
 * [Home :house_with_garden:](/)
+* Logic Apps
+  * [Controlling a single Azure Logic App](./logicapps/control-single-logicapp.md)
```
This gives a 404

stijnmoreels replied:
It doesn't for me 😅.

mbraekman replied:
It seems like you don't get a 404 when clicking the link when on the homepage, from any other page I'm getting the 404 as well.

tomkerkhove replied:
This might help you @stijnmoreels - https://github.com/invictus-integration/docs-ifa/blob/master/_sidebar.md

stijnmoreels replied:
I removed the dot `.` at the beginning. That seems the only difference.

tomkerkhove commented on docs/_sidebar.md:
```diff
@@ -1,3 +1,6 @@
 * [Home :house_with_garden:](/)
+* Logic Apps
+  * [Controlling a single Azure Logic App](./logicapps/control-single-logicapp.md)
+  * [Retrieving Azure Logic App runs by polling](./logicapps/polling-logicapp-runs.md)
```
This gives a 404

tomkerkhove commented:
Nitpick but let's use `logic-app` in URLs rather than `logicapp` to make it more readable since they are two separate words
